src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,52 @@
 
 class Animal {
   // write your code here
+  static alive = [];
+
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  }
+
+  die() {
+    const index = Animal.alive.indexOf(this);
+
+    if (index > -1) {
+      Animal.alive.splice(index, 1);
+    }
+  }
+
+  static updateAlive() {
+    Animal.alive = Animal.alive.filter((animal) => animal.health > 0);
```

Comment:
The `updateAlive` static method correctly uses the `filter` method to remove animals with health less than or equal to 0 from the `Animal.alive` array. This is a good practice and adheres to the checklist requirement.